stdlib/std.ncl

```diff
@@ -815,8 +815,7 @@
                     (
                       fun field =>
                         contract_map."%{field}"
-                    ),
-            `Array =>
+                    ),`Array =>
```

Comment:
This formatting change is fishy.

Reply:
There seems to be some edge case with topiary? There was a similar formatting issue in another PR today.

Reply:
https://github.com/tweag/topiary/issues/439